packages/async-queue/cliff.toml

```diff
@@ -24,7 +27,10 @@ body = """
 		- {% if commit.scope %}\
 			**{{commit.scope}}:** \
 		  {% endif %}\
-            {{ commit.message | upper_first }} ([{{ commit.id | truncate(length=7, end="") }}](https://github.com/sapphiredev/utilities/commit/{{ commit.id }}))\
+            {{ commit.message | upper_first }} ([{{ commit.id | truncate(length=7, end="") }}]({{ self::remote_url() }}/commit/{{ commit.id }}))\
+            {% if commit.github.pr_number %} (\
+                [#{{ commit.github.pr_number }}]({{ self::remote_url() }}/pull/{{ commit.github.pr_number }})) \
+            {%- endif %}\
```

Comment:
Can we get the PR authors in this now?

Reply:
yes, pushed in [f19ff26](https://github.com/sapphiredev/utilities/pull/758/commits/f19ff264c77dc9bd0fd5e002b61213c4292542d2)